validator-api/Rocket.toml

```diff
@@ -0,0 +1,20 @@
+[development]
```

Comment:
Should this file be even committed by default?

Reply:
I've left it in as a sane default, we can remove it